sklearn/cluster/tests/test_optics.py

```diff
@@ -122,20 +122,7 @@ def test_extract_xi():
     clust = OPTICS(min_samples=2, min_cluster_size=2,
                    max_eps=np.inf, cluster_method='xi',
                    xi=0.04).fit(X)
-    assert_array_equal(clust.labels_, expected_labels)
-
-
-def test_cluster_hierarchy_():
```

Comment:
any particular reason you removed this test?

Reply:
this test fails if we change a random_state, and I guess it's not reasonable?